### PR TITLE
mock action performer configs

### PR DESCRIPTION
**Pull request by jeberl**
Summary
---------

Initial mock of the get_action_performer function. This will eventually read configs from s3 and dynamically determine which performer to use 

Test Plan
---------

1. go to https://webhook.site/ to get a unique URL
2.  add to the WebhookActionPerformer constructor
3. run `python3 -m hmalib.lambdas.actions.action_performer`
4. check that the webhook was recieved

### CODE REVIEW

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (Dcallies) [inline hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:36]: I built the dynamodb config thing for these types of usecases, probably overall easier for same general results.

File: hasher-matcher-actioner/hmalib/common/actioner_models.py
Comment (Dcallies) [inline hasher-matcher-actioner/hmalib/common/actioner_models.py:43]: There might be a specific type for this buried in the typing module

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (Dcallies) [inline hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:35]: I like this approach, it's a great way to define the interfaces and unblock work, while someone else can iterate on the storage and config separately.

**Review by Dcallies — COMMENTED** [review]
I will leave approval and review to @eriksendc , I'm still struggling with the fact that I don't agree with the with the overall approach, and even though I had some suspicions originally it was because I didn't understand the proposal, after reading the code, I'm still hung on the same things.

 I know you are working with Brian on this, and I will defer to you to you two as the owners of the actioning to figure that out.

File: hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py
Comment (jeberl) [reply hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py:36]: Yeah. I figured I would implement once your PR merges

File: hasher-matcher-actioner/hmalib/common/actioner_models.py
Comment (eriksendc) [inline hasher-matcher-actioner/hmalib/common/actioner_models.py:60]: nix "unused argument" `_data`.

File: hasher-matcher-actioner/hmalib/common/actioner_models.py
Comment (eriksendc) [inline hasher-matcher-actioner/hmalib/common/actioner_models.py:72]: ditto

**Review by eriksendc — APPROVED** [review]
Approving so not blocked, but please make the three changes I requested.

File: hasher-matcher-actioner/hmalib/common/actioner_models.py
Comment (jeberl) [reply hasher-matcher-actioner/hmalib/common/actioner_models.py:43]: Couldn't find

**Review by eriksendc — APPROVED** [review]

**Pull request merged**
